definitions/infra-azureapimanagementservice/summary_metrics.yml

```diff
@@ -11,7 +11,7 @@ requests:
     select: sum(totalRequests.Total)
     from: AzureApiManagementServiceSample
 averageResponseTimeS:
-  title: Average response time (s)
+  title: Average response time (ms)
   unit: SECONDS
```

Comment:
It isn't clear to me what we should do with the unit when the metric is in milliseconds. Should we divide the value by 1000 in the query and leave the metric in seconds?

Reply:
Yes we should. In fact the UI adapts to whatever it seems fit.

So if the value is 0.05s it will probably show 5ms which makes me thing that the unit in the title is probably not a good thing.. but that's probably for another PR/discussion

Reply:
My knee-jerk reaction is that it'd be nice for all entity types to report times in the same unit, but at the same time I can see this being impractical (e.g. one entity type having response times where the reasonable unit can be ms, another entity type having a different time metric where ms is too small).  For now it seems ok that we optimize locally to each type. WDYT?

Reply:
What should we put as unit in that case? I don't think we expose `MILLISECONDS` as a unit and having to specify in the title seems pretty hacky.

Reply:
it won't always be displayed in ms in the UI as it changes based on the value. e.g. 1000 ms might become 1s. I would not specify the unit in the title